documentation/modules/managing/proc-manual-rolling-update-pods.adoc

```diff
@@ -8,6 +8,34 @@
 This procedure describes how to manually trigger a rolling update of an existing Kafka cluster or ZooKeeper cluster using a Kubernetes `Pod` annotation.
 When multiple pods from the same `StatefulSet` are annotated, consecutive rolling updates are performed within the same reconciliation run.
 
+.Prerequisites
+
+You can perform a rolling update on a Kafka cluster regardless of the topic replication factor used.
+But for Kafka to stay operational during the update, you'll need the following:
+
+* A highly available Kafka cluster deployment running with nodes that you wish to update.
```

Comment:
I think "wish" tends towards idiomatic. I think "would like to" might be a possible reword.

Reply:
_would like_ sounds good to me